models/shipment.rb

```diff
@@ -37,7 +37,7 @@ class Shipment < Base
     # validates_numericality_of :visits_number
 
     field :unavailable_visit_indices, default: []
-    field :unavailable_visit_day_indices, default: [] # extends unavailable_visit_day_date
+    field :unavailable_days, default: [] # extends unavailable_visit_day_date and unavailable_visit_day_indices
```

Comment:
It may be interesting to have a Set instead of an array. The order doesn't really matter, we only want to know if a value is included ? `include?` method of a Set is in O(1) against O(n) for an Array